lib/tasks/load_stocks.rake

```diff
@@ -0,0 +1,9 @@
+require 'csv'
+
+task load_stocks: :environment do
+  table = CSV.parse(File.read('./public/stocks.csv'), headers: true)
+
+  table.each do |stock|
+    Stock.create({ name: stock[0], ticker: stock[1]})
+  end
+end
```

Comment:
can you change it so that the rake task takes in an argument for how many records to load into DB? I don't want to wait the whole day to populate the table.
If argument (number of records) is passed the task will use that, otherwise it will load everything.

Reply:
it take less than a minute.